bc_obps/registration/constants.py

```diff
@@ -29,4 +29,5 @@
 USER_OPERATOR_TAGS_V2 = ["User Operator V2"]
 MISC_TAGS = ["Misc V1"]
 CONTACT_TAGS = ["Contact V1"]
+TRANSFER_EVENT_TAGS = ["Transfer Event V1"]
```

Comment:
I think this is correct--even though we're in reg2, we didn't have events in reg2, so I modelled this after facility